pb/config/config.proto

```diff
@@ -294,10 +292,15 @@ message TestGroup {
   // methods.
   bool link_bugs_by_test_methods = 45;
 
-  // Regex to match test methods. Only tests methods with names that match
+  // Regex to match test methods. Only test methods with names that match
   // this regex will be included in the table.
   string test_method_match_regex = 46;
 
+  // Regex to exclude test methods. Test methods with names that match
+  // this regex will be excluded from the table, even if they match
+  // test_method_match_regex.
+  string test_method_unmatch_regex = 59;
```

Comment:
This number will need to be 61; it looks like 59 and 60 are reserved internally.

Reply:
xref: https://github.com/GoogleCloudPlatform/testgrid/issues/26

Reply:
Done